benefits/eligibility/views.py

```diff
@@ -112,10 +112,10 @@ def start(request):
         )
 
         if not session.logged_in(request):
-            button = viewmodels.Button.external(
-                text=_(verifier.auth_provider.sign_in_button_label),
+            button = viewmodels.Button.login(
+                label=_(verifier.auth_provider.sign_in_button_label),
+                text="",
                 url=reverse("oauth:login"),
```

Comment:
Since this is specifically the `login` button, maybe move this `url=` to the `Button.login()` helper? Same as how the `id` was moved in there, it's part of how the button is defined right?

Reply:
I thought about that, but for https://github.com/cal-itp/benefits/pull/493, I want to be able to make a Log _out_ button with the same method too, so I opted to keep `url` defined out of the method.

Reply:
That makes way more sense, thank you! 🙏